### PR TITLE
Various improvements added

### LINKED ISSUE #44: Various improvements
There are certain bugs which can be fixed in the eos-icons website.
1.The text "Implementation example" is mentioned twice. We can make it show only one.

2. These are some typos which can be fixed.
image_2022-02-18_141400

3. We can add the GitHub repository link just after the "Go to our GitHub to find out more"

4. There is a broken link to Saurabh Sharma Sir's twitter account(saying the account does not exist when we open it). We can add the updated link.

5. Fix text and link mobile responsiveness issue(as here we can see colon is below with the link not with the text)

### PR DESCRIPTION
**Pull request by AnshumanDhiman**
Fixed #44 Added various improvements fixing typos, dead link and various responsiveness issues.

### CODE REVIEW

File: src/pages/Docs.js [hunk 1]
```diff
@@ -165,7 +165,8 @@ const Docs = () => {
                 <code>gem install eos-icons-font</code>
               </pre>
               <p>
-                Add the following directive to your application:{' '}
+                Add the following directive to your application:
```
Comment (vinayaksh42) [inline src/pages/Docs.js:168]: For the desktop view, the `*= require...` line should come right after the `:`
opera_C5hD3W6YFO

File: src/pages/Docs.js [hunk 2]
```diff
@@ -295,6 +295,7 @@ const Docs = () => {
               <p>
                 EOS icons is open source. Go to our Gitlab repository to find
                 out more :
+                <br />
```
Comment (vinayaksh42) [inline src/pages/Docs.js:298]: For the desktop view, the `https://gitlab.com/SUSE-UIUX/eos-icons` line should come right after the `:`
opera_3f04j5566k

File: src/pages/Docs.js [hunk 3]
```diff
@@ -689,17 +690,19 @@ export default App;`}</code>
...
-                  data-event-category='External link'
-                  data-event-action='Link to Github repository'
-                  data-event-label='Docs page'
-                  target='_blank'
-                  rel='noopener noreferrer'
-                >
-                  {' '}
-                  https://github.com/EOS-uiux-Solutions/eos-icons-react
-                </a>
+                <div className='resp-link-edit'>
```
Comment (vinayaksh42) [inline src/pages/Docs.js:693]: For the desktop view, the `https://github.com/EOS-uiux-Solutions/eos-icons-react` line should come right after the `:`
opera_9NsTAZtUo2

**Review by vinayaksh42 — CHANGES_REQUESTED** [review]

**Comment by cyntss** [comment]
@vinayaksh42 can you review the last changes?

**Review by vinayaksh42 — APPROVED** [review]
Great work @AnshumanDhiman

**Pull request merged**